src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/query/CypherExpression.java

```diff
@@ -16,4 +18,8 @@ public interface CypherExpression {
      */
     Set<CypherVar> getVars();
 
+    /**
+     * accepts a {@link CypherExpressionVisitor} and propagates the visit recursively when required.
+     */
+    void acceptVisitor(final CypherExpressionVisitor visitor);
```

Comment:
The function name that I have used for the visitor pattern in other parts of the code base is `visit`. Unless there is a strong reason for using `acceptVisitor` here, can you please rename this function to `visit` for the sake of consistency.

Reply:
I had forgotten we used visitors elsewhere, so I just named the methods as they made more sense to me (i.e.,  the expressions receive a visitor), but I have renamed it to be consistent.